client/src/components/Draftail/index.js

```diff
@@ -205,8 +211,12 @@ const initEditor = (selector, originalOptions, currentScript) => {
         </>
       ),
       bottomToolbar: MetaToolbar,
-      commandPalette: (props) => (
-        <CommandPalette {...props} noResultsText={gettext('No results')} />
+      commandToolbar: (props) => (
```

Comment:
`commandPalette` was never the correct prop 😭 Until now the consequence was only the "no results text" being wrong and not translate-able.